app/setup_handlers.go

```diff
@@ -36,6 +39,20 @@ func (app *ElysApp) setUpgradeHandler() {
 				}
 				consensusParams.Block.MaxBytes = NewMaxBytes
 				app.ConsensusParamsKeeper.ParamsStore.Set(ctx, consensusParams)
+
+				// Iterate over all the keys in the wasm module store
+				// and delete them
+				// TODO: Delete wasm code after deleting wasm module store
+				// Retrieve the wasm module store key
+				storeKey := app.GetKey(wasmtypes.StoreKey)
```

Comment:
If you use the actual string instead of the const from the wasm module, wouldn't that allow you to also remove the wasm module files, same goes for deleting the entire wasm module store below

Reply:
I was thinking that but wasmmodule is closely integrated within app,wasmkeeper and wasmopts are passed in app initialisation, so was thinking to remove these states first and then remove whole everything

Reply:
Sounds good let do that first